.github/workflows/build.yml

```diff
@@ -0,0 +1,24 @@
+name: Prettier on Push and PR
+
+on:
+    push:
+        branches:
+            - 'main'
+            - 'dev**'
+    pull_request:
+        branches:
+            - 'main'
+
+jobs:
+    prettier:
+        runs-on: ubuntu-latest
+        
+        steps:
+            - uses: actions/checkout@v4
+            - uses: actions/setup-node@v4
+              with:
+                node-version: '18.x'
+            - name: Install dependencies
+              run: npm install

```

Comment:
Can we change this to yarn instead? Yarn is multithreaded and probably will make the install faster than `npm install`